### PR TITLE
Add RTL support for BpkIcon

### PR DESCRIPTION
**Pull request by jmkaluzka**
`withRtlSupport` will flip horizontally the icon if `I18nManager.isRTL`

### CODE REVIEW

File: packages/react-native-bpk-component-icon/README.md [hunk 1]
```diff
@@ -74,6 +74,38 @@ export default class App extends Component {
...
+import BpkIcon, { icons, withRtlSupport } from 'react-native-bpk-component-icon';
+import { spacingBase, colorBlue500 } from 'bpk-tokens/tokens/base.react.native';
+
+const BpkIconwithRtlSupport = withDivider(BpkIcon);
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,BpkIconwithRtlSupport
```
Comment (matthewdavidson) [inline packages/react-native-bpk-component-icon/README.md]:
```suggestion
    padding: spacingBase,
```

File: packages/react-native-bpk-component-icon/README.md [hunk 2]
```diff
@@ -74,6 +74,38 @@ export default class App extends Component {
...
 
+### `withRtlSupport` HOC
+
+```js
+import { View } from 'react-native';
+import React, { Component } from 'react';
+import BpkIcon, { icons, withRtlSupport } from 'react-native-bpk-component-icon';
+import { spacingBase, colorBlue500 } from 'bpk-tokens/tokens/base.react.native';
+
+const BpkIconwithRtlSupport = withDivider(BpkIcon);
```
Comment (matthewdavidson) [inline packages/react-native-bpk-component-icon/README.md]:
```suggestion
const BpkIconwithRtlSupport = withRtlSupport(BpkIcon);
```

**Comment by matthewdavidson** [comment]
Looks like we are missing an api token env variable on CI 🤔 see https://travis-ci.org/Skyscanner/backpack-react-native/builds/490632212#L524.

How come it still works for PR's from Skyscanner/backpack-react-native repo owners?

File: packages/react-native-bpk-component-icon/README.md (hunk 1 above)
Comment (jmkaluzka) [reply packages/react-native-bpk-component-icon/README.md]: Ups, looks like copy-paste mistake. Corrected

File: packages/react-native-bpk-component-icon/README.md (hunk 2 above)
Comment (jmkaluzka) [reply packages/react-native-bpk-component-icon/README.md]: Same

**Pull request merged**